doc/classes/bool.xml

```diff
@@ -4,7 +4,45 @@
 		Boolean built-in type.
 	</brief_description>
 	<description>
-		Boolean built-in type.
+		Boolean is a built-in type. It can represent any data type that is either a true or false value. You can think of it as on or off (1 or 0) switch. It's often used as part of programming logic in condition statements like [code]if[/code] statements.
```

Comment:
"think of it as **an** on or off (1 or 0) switch"

That's a lot of 2-letter words in a row, might not be easy to read :)

Reply:
Apart from the first line where "true" and "false" are used as adjectives, all other instances should use code markup with `[code]true[/code]` and `[code]false[/code]` since they refer to keywords.